### PR TITLE
feat: php docker page add extra callouts

### PR DESCRIPTION
**Pull request by ZNeumann**
Add more info to the php docker docs page. This consolidates some important information to a page where it will be easier to find.

### CODE REVIEW

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (mfulb) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:17]: I think you may mean "starting up" not "standing up"?

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (mfulb) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:66]: Maybe you could add a link to the recommended values?

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (zsistla) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:17]: please use: `New Relic` :)

Wording suggestion: The PHP Agent's daemon transmits data to New Relic periodically during the minute-long [harvest cycle](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/getting-started/glossary#harvest-cycle). If you are standing up and tearing down containers often, ensure that you leave the daemon container running long enough to transmit any remaining data.



(btw it's described in this page: https://docs.newrelic.com/docs/apm/transactions/transaction-traces/introduction-transaction-traces/)

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (zsistla) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:34]: suggestion: `If using short-lived application containers, it is recommended to use a separate container for the PHP Agent daemon.`

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (zsistla) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:66]: what are the recommended values?

**Comment by zsistla** [comment]
Great job, thanks for putting this up!

**Comment by paperclypse** [comment]
@ZNeumann When this is ready to be published, just click that "Ready for review" button and I (or someone else on the Docs team) will help you get this published.

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (paperclypse) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:17]:
```suggestion
 The PHP Agent's daemon transmits data to New Relic periodically during the minute-long [harvest cycle](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/getting-started/glossary#harvest-cycle). If you're starting up and tearing down containers often, ensure that you leave the daemon container running long enough to transmit any remaining data.
```

We generally want to use contractions.

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (paperclypse) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:34]:
```suggestion
To see an example application, go to [New Relic's Explorers Hub](https://discuss.newrelic.com/t/relic-solution-php-agent-and-daemon-containers/84841). If you're using short-lived application containers, we recommend you use a separate container for the PHP Agent's daemon.
```

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (paperclypse) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:66]:
```suggestion
By default, the first transaction causes the agent to trigger a daemon start and an application connection initialization. For performance reasons, the agent does not wait for those operations to complete before it initializes the connection. This can result in the loss of the first few transactions after a container starts. To prevent this loss, set both `newrelic.daemon.start_timeout` and `newrelic.daemon.app_connect_timeout` entries in the `newrelic.ini` file to our (recommended values)[https://discuss.newrelic.com/t/php-troubleshooting-framework-configuration/119980] of 5s and 15s respectively.
```

Cleaning up a bit of the wording here.

File: src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx
Comment (paperclypse) [inline src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx:106]:
```suggestion
  Here are some Docker troubleshooting resources:
```

**Review by paperclypse — APPROVED** [review]
Looks good!

**Comment by paperclypse** [comment]
@ZNeumann I made some minor edits for clarity. I'm happy to wait for you to review them or I can just merge this in once the checks are done. Thanks!

**Comment by ZNeumann** [comment]
@paperclypse Thank you! Everything looks good to me :)

**Pull request merged**
